content/powershell/concepts/variables/variables.md

```diff
@@ -1,12 +1,68 @@
 ---
 Title: 'Variables'
-Description: ''
+Description: 'Variables are a way to store and manipulate data. There are multiple types including environment variables which can be created in variety of ways.'
 Subjects:
+  - 'Code Foundations'
   - 'Computer Science'
+  - 'PowerShell'
 Tags:
   - 'Variables'
   - 'Data Types'
+  - 'Environment Variables'
+  - 'Variable Types'
 CatalogContent:
   - 'learn-powershell'
   - 'paths/computer-science'
 ---
+
+A **variable** is a location in computer memory used to store data, usually to be referred to and manipulated in a program. Variables in PowerShell can store the results of commands and expressions.
+
+## Using a Variable
+Variables in PowerShell are referenced using a dollar sign `$` followed by a valid variable name.
+```PowerShell
+PS > $pi
+3.14
+```
+
+## Creating a Variable
+To create a variable, a variable reference is followed by an equal sign `=` and the value to be assigned. 
+```PowerShell
+$my_string_variable = "Hello, World!"
+```
+
+Variables names are NOT case-sensitive and can include spaces and special characters. The convention is to use only alphanumeric characters and the underscore `_` character.
```

Comment:
```suggestion
Variables names are **not** case-sensitive and can include spaces and special characters. The convention is to use only alphanumeric characters and the underscore `_` character.
```

Reply:
Per [style-guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#supported-code-blocks), I used italics for emphasis in 2e4b8df.